spec/app/ics-031-crosschain-queries/README.md

```diff
@@ -299,14 +359,27 @@ function CrossChainQueryResponse(
     // Delete the query from the local, private store.
     privateStore.delete(queryPath(queryId))
 
+    // Reassign query's heightUpdate and value in case the query is of type PERSISTENT and the
+    // result was SUCCESS
+    heightUpdate = null
+    value = data
+    if (query.type === PERSISTENT && data !== null) {
```

Comment:
We also need to check that the previous value and height equal the previous resultRecord